.github/dependabot.yml

```diff
@@ -6,3 +6,9 @@ updates:
     interval: weekly
   open-pull-requests-limit: 10
   versioning-strategy: increase-if-necessary
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
+  open-pull-requests-limit: 10
+  versioning-strategy: increase-if-necessary
```

Comment:
@naveensrinivasan - It looks like `versioning-strategy` cannot be used with the `github-actions` update.

See the failed run here: https://github.com/openlayers/openlayers/runs/6203836160

Fix in #13614.

Reply:
Sorry about that. Thanks!